x-pack/legacy/plugins/reporting/server/routes/generation.ts

```diff
@@ -35,45 +35,64 @@ export function registerJobGenerationRoutes(
   async function handler(
     exportTypeId: string,
     jobParams: object,
-    legacyRequest: Legacy.Request,
-    h: ReportingResponseToolkit
+    r: RequestFacade,
+    h: typeof kibanaResponseFactory
   ) {
-    const request = makeRequestFacade(legacyRequest);
-    const user = request.pre.user;
-    const headers = request.headers;
+    const licenseInfo = reporting.getLicenseInfo();
+    const licenseResults = licenseInfo[exportTypeId];
+
+    if (!licenseResults.enableLinks) {
+      throw boom.forbidden(licenseResults.message);
```

Comment:
NIT: use `h.forbidden({body: licenseResults.message})`

Same in `handleError` function below.